source/materials/OpticalMaterialProperties.cc

```diff
@@ -1323,55 +1292,55 @@ G4MaterialPropertiesTable* OpticalMaterialProperties::LYSO()
-
-  G4double intensity[sc_entries] = {0., 0.0643, 0.1929, 0.2500, 0.4, 0.5214, 0.9571,
-                                    1, 0.8286, 0.6, 0.4071, 0.2, 0.};
-
+  const G4int sc_entries = 63;
+
+ G4double sc_energy[] =
+   {1.9629408004008015*eV, 1.9908574214449701*eV, 2.017487971587794*eV, 2.049118407065618*eV,
+    2.075153360986118*eV, 2.1108921203127564*eV, 2.131530815691394*eV, 2.1549516183402426*eV,
+    2.1788791714153226*eV, 2.2008686890176556*eV, 2.225818446565337*eV, 2.2461704051922378*eV,
+    2.2774394451688957*eV, 2.2933838194379406*eV, 2.309537681443672*eV, 2.3287011454278184*eV,
+    2.34534235398821*eV, 2.365099103796332*eV, 2.382250183859144*eV, 2.4085359216953424*eV,
+    2.4233406375845*eV, 2.456622921785144*eV, 2.4782281493172356*eV, 2.503411781001508*eV,
+    2.52588828700936*eV, 2.5487533714262676*eV, 2.572055225192039*eV, 2.588926183988832*eV,
+    2.6025967807599906*eV, 2.623400276029348*eV, 2.6374283564632037*eV, 2.6588048713474666*eV,
+    2.6623164236046244*eV, 2.6877929388662145*eV, 2.713729981831732*eV, 2.7325282355525173*eV,
+    2.7593839494903407*eV, 2.7907427243849288*eV, 2.83096946231753*eV, 2.885218973585188*eV,
+    2.9197414497478893*eV, 2.9417873478198326*eV, 2.9732295992116424*eV, 2.991505736849123*eV,
+    3.005429157443019*eV, 3.0194959021101067*eV, 3.0242886485355824*eV, 3.0432131984421833*eV,
+    3.048094952090428*eV, 3.057784590715629*eV, 3.0675360309025064*eV, 3.0870759513756334*eV,
+    3.1271371999622284*eV, 3.1373650393337758*eV, 3.1527828353680922*eV, 3.1734564909877436*eV,
+    3.189158818959311*eV, 3.199797076377322*eV, 3.2210492254756646*eV, 3.242646039884366*eV,
+    3.2699528217360423*eV, 3.2976765066977722*eV, 3.3428842485014827*eV};
+
+
+ G4double intensity[sc_entries] = {0., 0.0704225352112644, 0.3521126760563362, 0.633802816901401,
+0.7746478873239369, 1.1971830985915446, 1.5492957746478881, 1.8309859154929529, 2.2535211267605604,
+2.676056338028168, 3.2394366197183047, 3.8732394366197203, 4.507042253521121, 5.0,
+5.633802816901407, 6.126760563380272, 6.901408450704224, 7.464788732394362, 8.38028169014084,
+9.014084507042256, 9.788732394366193, 10.985915492957744, 12.183098591549294, 13.38028169014084,
+14.29577464788732, 15.352112676056334, 16.267605633802816, 17.183098591549292, 17.816901408450704,
+18.591549295774648, 19.295774647887324, 20.0, 20.704225352112672, 21.338028169014088,
+22.183098591549296, 23.02816901408451, 23.59154929577465, 24.15492957746479, 24.788732394366196,
+24.507042253521124, 23.80281690140845, 23.09859154929577, 22.04225352112676, 21.408450704225352,
+20.56338028169014, 19.64788732394366, 18.943661971830984, 18.239436619718308,
+17.464788732394368, 16.549295774647888, 15.633802816901408, 14.577464788732392, 11.338028169014079,
+10.281690140845065, 8.943661971830984, 7.746478873239432, 6.760563380281688, 5.7042253521126725,
+4.718309859154921, 3.450704225352105, 2.3239436619718252, 1.4084507042253518, 0.};
```

Comment:
Why the maximum intensity is close to 25 while in the previous implementation it was normalized to 1?

Reply:
The absolute maximum value is not relevant, since GEANT4 normalizes the distribution anyway. The relative intensities are important.

Reply:
I see, and where do these values come from? Maybe I am missing the reference or rather does it come from a simulation?

Reply:
I've added the reference to the SaintGobain company datasheet.